package.json

```diff
@@ -24,7 +24,7 @@
     "build-lr": "yarn reset-link && node ./build/build-lightrider-bundles.js",
     "build-pack": "bash build/build-pack.sh",
     "build-report": "node build/build-report-components.js && yarn eslint --fix report/renderer/components.js && node build/build-report.js",
-    "build-sample-reports": "yarn build-report && node build/build-sample-reports.js",
+    "build-sample-reports": "yarn build-report && node build/build-sample-reports.js && node build/build-css-class-list.js",
```

Comment:
remove from this script, and instead change `build-lr` to be

```
yarn reset-link && node ./build/build-lightrider-bundles.js && yarn build-sample-reports && node build/build-css-class-list.js
```

then the roll scripts in google3 dont need to change which yarn scripts it runs.